Formula/webdis.rb

```diff
@@ -0,0 +1,37 @@
+class Webdis < Formula
+  desc "Redis HTTP interface with JSON output"
+  homepage "http://webd.is/"
+  url "https://github.com/nicolasff/webdis/archive/0.1.2.tar.gz"
+  sha256 "8e46093af006e35354f6b3d58a70e3825cd0c074893be318f1858eddbe1cda86"
+
+  depends_on "libevent"
+
+  def install
+    system "make"
+    bin.install "webdis"
+
+    inreplace "webdis.prod.json" do |s|
+      s.gsub! "/var/log/webdis.log", "#{var}/log/webdis.log"
+    end
+
+    etc.install "webdis.json"
+    etc.install "webdis.prod.json"
+  end
+
+  test do
+    # Based on dcd formula test. Thanks @John-Colvin!
+    # https://github.com/Homebrew/homebrew-core/blob/b9dd5222a93f4f843e766b94d52748d633c4e0aa/Formula/dcd.rb#L32
+    begin
+      server = fork do
+        exec "#{bin}/webdis", "#{etc}/webdis.json"
+      end
+      # Give it generous time to load
+      sleep 0.5
+      # Test that the response is from Webdis (the request may not be successful if Redis is not running)
```

Comment:
webdis does not require Redis to be installed on the machine. For correct operation, webdis does require config pointing to a running Redis instance.